website/main/content/blog/2025-07-14-scheduled-plot-lifecycle.md

```diff
@@ -0,0 +1,221 @@
+---
+title: A Deterministic Mapping for Plot Lifecycle Management
+date: 2025-07-14
+draft: false
+description: A new model for plot and sector lifecycle management and a draft spec in progress
+tags: [status-update, consensus]
+authors: [adlrocha]
+---
+
+{{< katex >}}
+
+This week has been mainly focused on refining a bit the design for plot identification and sector
+expiration. I think that I finally have a model with which I am comfortable with, and that I think
+solves all of our previous problems. Nazar had this idea to drastically simplify how plot IDs were
+derived, and how sectors were linked to plots. The high-level idea made sense, but there were still
+some details that weren't clear. This week I managed to come up with a design that I think satisfies
+all of our requirements.
+
+Apart from all of this work around plot and sector lifecycle management, I also started working on a
+draft specification for the protocol that fits all the pieces together. Our protocol builds upon the
+Subspace protocol, so I am using the existing specification as a base and extending it with all the
+sharding specifics. I can't wait to share the first draft with you all for feedback and suggestions.
+
+## A Deterministic Mapping for Plot Lifecycle Management
+
+The core of this proposal is based on the simplification idea for sector expiration and plot
+identification from this [zulip discussion]. The basic idea is to simplify plot identification by
```

Comment:
`[zulip discussion]` doesn't have a link corresponding to it.

Reply:
It was lost in one of my edits for some reason. Adding it at the end of the doc.